source/reference/minio-mc-admin/mc-admin-user.rst

```diff
@@ -22,10 +22,10 @@ The :mc:`mc admin user` command and its subcommands manage :ref:`MinIO users <mi
 Clients *must* authenticate to the MinIO deployment with the access key and secret key associated to a user on the deployment.
 MinIO users constitute a key component in MinIO Identity and Access Management.
 
-To manage users who authenticate using a 3rd party IDP, use the :mc:`mc admin` commands for the appropriate provider:
+To manage users who authenticate using a 3rd party IDP, use the :mc:`mc` commands for the appropriate provider:
 
-- For AD/LDAP, use :mc:`mc admin idp ldap`
-- For OpenID Connect (OIDC) compatible providers, use :mc:`mc admin idp openid`
+- For AD/LDAP, use :mc:`mc idp ldap`
+- For OpenID Connect (OIDC) compatible providers, use :mc:`mc idp openid`
 
 .. admonition:: Use ``mc admin`` on MinIO Deployments Only
```

Comment:
Is this admonition still accurate now that these commands have moved out of `mc admin`?

Reply:
Hm. 

Lets confirm with engineering, but I have to assume yes. The key is that these are designed to work against a MinIO deployment. We have no way to know how the admin API for some other S3-compatible service would work.

In that case the admonition should simply read "Use for MinIO Deployment Only"

Reply:
Would need a new include too. Can just add a new block of text in a start-end fence stating that 'This command is only supported against MinIO deployments.'

Reply:
I tried an include that worked in the needed locations and wasn't happy with it. (Specifically for `mc admin user` but also meh about the `idp` ones.) Added appropriate text for each.

Another thing to consider when these are converted to individual subcommand pages.